src/channel.cr

```diff
@@ -114,6 +114,10 @@ class Channel(T)
     end
   end
 
+  # Closes the channel.
+  # The method prevents any new value from being sent to / received from the channel.
+  # It wakes up any sender / receiver fibers waiting on the channel.
+  # It only has effect *after* any previously enqueued value in the channel has been received, up to the channel's capacity.
```

Comment:
I would add that calling close on a closed channel does not have any effect, but such behaviour hasn't been documented in the specs, so I'd better add a test too, if we're keen on being explicit on that

Reply:
I'd document and spec that. You can do that in this PR.

Reply:
Done:

+ added to `Channel#close` docs
```
Calling `#close` on a closed channel does not have any effect.
```
+ Added spec
```
Channel "does not raise nor change its status when it is closed more than once"
```